scripts/caches-check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# script to manually call vmaas_sync
+
+curl http://localhost:9999/caches-check
```

Comment:
I'd name it check-caches, since it is an action performed, not a resource.

Reply:
fixed